README.md

```diff
@@ -105,6 +105,13 @@ The same options available on React Native's [ActionSheetIOS](https://facebook.g
 #### `anchor` (optional)
 iPad only option that allows for docking the action sheet to a node. See [ShowActionSheetButton.tsx](/example/ShowActionSheetButton.tsx) for an example on how to implement this.
 
+| Name               | Type   | Required | Default |
+| -------------------| -------| -------- | ------- |
+| userInterfaceStyle | string | No       |         |
```

Comment:
Can you add this to the table above please? Thank you!

Reply:
Oops, I need more coffee :) Fixed!